### PR TITLE
Notice gen viola&Recv commit

### PR DESCRIPTION
**Pull request by kshitijzutshi**
Notice gen viola&Recv flow commit

### CODE REVIEW

File: web/rainmaker/dev-packages/egov-rented-properties-dev/src/ui-config/screens/specs/rented-properties/applyResource/footer.js [hunk 1]
```diff
@@ -160,6 +174,119 @@ const callBackForNext = async(state, dispatch) => {
     }
 }
 
+const callBackForNextrecoveryNoticegeneration = async(state, dispatch) => {
+
+  let uploadFlag = false;
+  let activeStep = 1;
+  let isFormValid = true;
+
+if(activeStep === 1){
```
Comment (tejaswi-transerve) [inline web/rainmaker/dev-packages/egov-rented-properties-dev/src/ui-config/screens/specs/rented-properties/applyResource/footer.js:183]: what is the need to active step here?

**Comment by kshitijzutshi** [comment]
Commited movetosuccess change

**Comment by kshitijzutshi** [comment]
changes made to mmove2success..

**Comment by kshitijzutshi** [comment]
purposetransit added

**Pull request merged**